casper/contracts/simple_casper.v.py

```diff
@@ -476,6 +497,8 @@ def slash(vote_msg_1: bytes <= 1024, vote_msg_2: bytes <= 1024):
     validator_deposit: num(wei) = self.get_deposit_size(validator_index_1)
     slashing_bounty: num(wei) = validator_deposit / 25
     self.total_destroyed += validator_deposit * 24 / 25
+    # Log slashing
+    log.Slash(msg.sender, self.validators[validator_index_1].withdrawal_addr, slashing_bounty)
```

Comment:
This is currently just logging the bounty (4%) finder's fee to whomever submit the slashing condition. Should we be logging the entirety (`validator_deposit`)?

Reply:
Good catch, lets log both and add validator index for uniformity.